src/components/molecules/cta/cta.module.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .ctaBg {
-  background-color: #085A7D;
+  background-color: #085a7d;
```

Comment:
Guessing these are all linter changes and we don't want assign variables yet?

Reply:
Correct on both accounts.

Reply:
I'm working on defining/using variables as I go through and update/create the components. So all of these changes are just from the linter auto-fixing stuff.